### DISCUSSION TITLE
Request for Assistance Regarding Delay in GitHub Education Benefits Disbursement

### DISCUSSION THREAD

**Discussion by sai-kashyap** [community/community#138464] [GitHub Education]
### Hi everyone,

I have not received my education benefits within 5 days of my approval(9th September 2024) . Please let me know what I can do expedite the processes
Thank you for your attention to this request.

**Comment by davevad93** [comment]
Hi @sai-kashyap ,

Don't worry you don't have to do anything to expedite the process, you'll have your benefits soon, it usually takes a few days but the waiting time may vary depending on different factors, such as the volume of requests, in which case it may take longer even a couple of weeks. 

 When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @sai-kashyap was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by sai-kashyap** [reply]
Thank You

**Reply by davevad93** [reply]
You're welcome @sai-kashyap 👍
